__tests__/index.spec.js

```diff
@@ -252,26 +255,45 @@ describe('one-app-dev-cdn', () => {
           expect(response.status).toBe(200);
           expect(response.header['content-type']).toMatch(/^application\/json/);
           const responseBody = response.text;
-          expect(responseBody).toBe(
-            JSON.stringify({ ...defaultRemoteMap, key: 'not-used-in-development' })
-          );
+
           expect(responseBody.modules).not.toEqual(defaultLocalMap.modules);
         });
     });
 
-    it('mirrors the remote map but modifies the key property', () => {
+    it('mirrors the remote map but modifies the key property and the module URLs', () => {
       expect.assertions(4);
       const remoteModuleMapUrl = 'https://my-domain.com/map/module-map.json';
       const fcdn = setupTest({ appPort: 3000, useLocalModules: false, remoteModuleMapUrl });
       got.mockReturnJsonOnce(defaultRemoteMap);
 
+      const modifiedRemoteMap = {
+        key: 'not-used-in-development',
+        modules: {
+          'module-b': {
+            node: {
+              url: 'http://0.0.0.0:3001/static/cdn/module-b/1.0.0/module-b.node.js',
+              integrity: '123',
+            },
+            browser: {
+              url: 'http://0.0.0.0:3001/static/cdn/module-b/1.0.0/module-b.browser.js',
+              integrity: '234',
+            },
+            legacyBrowser: {
+              url: 'http://0.0.0.0:3001/static/cdn/module-b/1.0.0/module-b.legacy.browser.js',
+              integrity: '345',
+            },
+          },
+        },
+      };
       return supertest(fcdn)
         .get('/module-map.json')
```

Comment:
can you test that if this is hit multiple times that the module map list does not keep growing